images/win/scripts/SoftwareReport/SoftwareReport.Android.psm1

```diff
@@ -1,27 +1,24 @@
-function Get-AndroidComponentLocation {
+function Split-TableRowByColumns {
     param(
-        [string] $ComponentName
+        [string] $Row
     )
-    $path = Join-Path $env:ANDROID_HOME $ComponentName
-    return "Location $path"
+    return $Row.Split("|") | ForEach-Object { $_.trim() }
 }
 
-function Split-AndroidSDKOutputRow {
+function Get-AndroidComponentLocation {
     param(
-        [string] $Row
+        [string] $ComponentName
     )
-    return $Row.Split("|").Trim()
+    $path = Join-Path $env:ANDROID_HOME $ComponentName
+    return "Location $path"
 }
 
-function Create-AndroidTableObject {
+function Get-AndroidSDKRoot {
```

Comment:
Looks like the same as Get-AndroidComponentLocation

Reply:
Removed it